engines/sherlock/people.cpp

```diff
@@ -0,0 +1,705 @@
+	{ "BELL" },			// Belle (perfumerie)
+	{ "GIRL" },			// Cleaning Girl (perfumerie)
+	{ "EPST" },			// Epstien in the Opera Balcony
+	{ "WIGG" },			// Wiggins
+	{ "PAUL" },			// Paul (Brumwell / Carroway)
+	{ "BART" },			// Bartender
+	{ "DIRT" },			// Dirty Drunk
+	{ "SHOU" },			// Shouting Drunk
+	{ "STAG" },			// Staggering Drunk
+	{ "BOUN" },			// Bouncer
+	{ "SAND" },			// James Sanders - At Home
+	{ "CORO" },			// The Coroner
+	{ "EQUE" },			// The Equestrian Shop Keeper
+	{ "GEOR" },			// George Blackwood
+	{ "LARS" },			// Lars
+	{ "PARK" },			// Sheila Parker (happy)
+	{ "CHEM" },			// Chemist
+	{ "GREG" },			// Inspector Gregson
+	{ "LAWY" },			// Jacob Farthington Lawyer
+	{ "MYCR" },			// Mycroft
+	{ "SHER" },			// Old Sherman
+	{ "CHMB" },			// Richard Chemist Stock boy
+	{ "BARM" },			// Barman
+	{ "DAND" },			// Dandy Player
+	{ "ROUG" },			// Rough-looking Player
+	{ "SPEC" },			// Spectator
+	{ "HUNT" },			// Robert Hunt
+	{ "VIOL" },			// Violet Secretary
+	{ "PETT" },			// Pettigrew
+	{ "APPL" },			// Augie (apple seller)
+	{ "ANNA" },			// Anna Carroway
+	{ "GUAR" },			// Guard
+	{ "ANTO" },			// Antonio Caruso
+	{ "TOBY" },			// Toby the Dog
+	{ "KING" },			// Simon Kingsley
+	{ "ALFR" },			// Alfred Tobacco Clerk
+	{ "LADY" },			// Lady Brumwell
+	{ "ROSA" },			// Madame Rosa
+	{ "LADB" },			// Lady Brumwell
+	{ "MOOR" },			// Joseph Moorehead
+	{ "BEAL" },			// Mrs. Beale
+	{ "LION" },			// Felix the Lion
+	{ "HOLL" },			// Hollingston
+	{ "CALL" },			// Constable Callaghan
+	{ "JERE" },			// Sergeant Jeremy Duncan
+	{ "LORD" },			// Lord Brumwell
+	{ "NIGE" },			// Nigel Jameson
+	{ "JONA" },			// Jonas (newspaper seller)
+	{ "DUGA" },			// Constable Dugan
+	{ "INSP" }			// Inspector Lestrade (Scotland Yard)
+};
+
+const char *const NAMES[MAX_PEOPLE] = {
+	"Sherlock Holmes",
+	"Dr. Watson",
+	"Inspector Lestrade",
+	"Constable O'Brien",
+	"Constable Lewis",
+	"Sheila Parker",
+	"Henry Carruthers",
+	"Lesley",
+	"An Usher",
+	"An Usher",
+	"Fredrick Epstein",
+	"Mrs. Worthington",
+	"The Coach",
+	"A Player",
+	"Tim",
+	"James Sanders",
+	"Belle",
+	"Cleaning Girl",
+	"Fredrick Epstein",
+	"Wiggins",
+	"Paul",
+	"The Bartender",
+	"A Dirty Drunk",
+	"A Shouting Drunk",
+	"A Staggering Drunk",
+	"The Bouncer",
+	"James Sanders",
+	"The Coroner",
+	"Reginald Snipes",
+	"George Blackwood",
+	"Lars",
+	"Sheila Parker",
+	"The Chemist",
+	"Inspector Gregson",
+	"Jacob Farthington",
+	"Mycroft",
+	"Old Sherman",
+	"Richard",
+	"The Barman",
+	"A Dandy Player",
+	"A Rough-looking Player",
+	"A Spectator",
+	"Robert Hunt",
+	"Violet",
+	"Pettigrew",
+	"Augie",
+	"Anna Carroway",
+	"A Guard",
+	"Antonio Caruso",
+	"Toby the Dog",
+	"Simon Kingsley",
+	"Alfred",
+	"Lady Brumwell",
+	"Madame Rosa",
+	"Lady Brumwell",
+	"Joseph Moorehead",
+	"Mrs. Beale",
+	"Felix",
+	"Hollingston",
+	"Constable Callaghan",
+	"Sergeant Duncan",
+	"Lord Brumwell",
+	"Nigel Jaimeson",
+	"Jonas",
+	"Constable Dugan",
+	"Inspector Lestrade"
+};
```

Comment:
Nitpicking: wouldn't it be better to merge these two arrays into one? You're holding one attribute of each person in different arrays.

i.e. you could create a structure:
    struct PersonData {
      const char *portrait;
      const char * name;
    };

And then merge the arrays like this:

    const PersonData people[MAX_PEOPLE] {
      { "HOLM", "Sherlock Holmes" },
      { "WATS", "Dr. Watson" },
      { "LEST", "Inspector Lestrade" },
      { "CON1", "Constable O'Brien" },
      { "CON2", "Constable Lewis" }
      (......)
    };